code/modules/surgery/xeno.dm

```diff
@@ -0,0 +1,223 @@
+
+//Research stuff to extract stuff from xenomorphs for goodies. In other words, to extract usefull material that could be used to upgrade marines and etc.
+
+/datum/surgery/xenomorph
+	name = "Experimental Harvesting Surgery"
+	invasiveness = list(SURGERY_DEPTH_SURFACE)
+	required_surgery_skill = SKILL_SURGERY_TRAINED
+	possible_locs = list("head")
+	target_mobtypes = list(/mob/living/carbon/xenomorph)
+	steps = list(
+		/datum/surgery_step/xenomorph/cut_exoskeleton,
+		/datum/surgery_step/xenomorph/open_exoskeleton,
+		/datum/surgery_step/xenomorph/severe_connections,
+		/datum/surgery_step/xenomorph/remove_organ,
+	)
+	lying_required = FALSE
+	requires_bodypart_type = NONE
+	requires_bodypart = FALSE
+
+/datum/surgery/xenomorph/can_start(mob/user, mob/living/carbon/xenomorph/patient, obj/limb/L, obj/item/tool)
+	if(islarva(patient) || isfacehugger(patient))
+		to_chat(user, SPAN_DANGER("This race is probably too small to have a mature organ worthy to extract..."))
+		return FALSE
+	if(patient.tier > 2 || isqueen(patient) && !istype(tool, /obj/item/tool/surgery/scalpel/laser/advanced))
+		to_chat(user, SPAN_DANGER("Chitin of this kind is too thick for an ordinary tool, you would need something special."))
+		return FALSE
+	if(patient.stat == DEAD && !patient.organ_removed)
+		return TRUE
+	return FALSE
+
+/datum/surgery_step/xenomorph/cut_exoskeleton
+	name = "Cut Exoskeleton Carapace"
+	desc = "cut the carapace open"
+	tools = list(\
+	/obj/item/tool/surgery/scalpel/laser/advanced = 1.5,\
+	/obj/item/tool/surgery/circular_saw = SURGERY_TOOL_MULT_IDEAL,\
+	/obj/item/weapon/twohanded/fireaxe = SURGERY_TOOL_MULT_SUBOPTIMAL,\
+	/obj/item/weapon/sword/machete = SURGERY_TOOL_MULT_SUBOPTIMAL,\
+	/obj/item/tool/hatchet = SURGERY_TOOL_MULT_SUBSTITUTE,\
+	/obj/item/tool/kitchen/knife/butcher = SURGERY_TOOL_MULT_SUBSTITUTE,\
+	/obj/item/attachable/bayonet = SURGERY_TOOL_MULT_BAD_SUBSTITUTE\
+	)
+
+	time = 4 SECONDS
+	preop_sound = 'sound/handling/clothingrustle1.ogg'
+	success_sound = 'sound/handling/bandage.ogg'
+	failure_sound = 'sound/surgery/organ2.ogg'
+
+/datum/surgery_step/xenomorph/cut_exoskeleton/preop(mob/living/carbon/human/user, mob/living/carbon/xenomorph/target, target_zone, obj/item/tool, tool_type, datum/surgery/surgery)
+	if(tool_type == /obj/item/tool/surgery/circular_saw)
+		user.affected_message(target,
+			SPAN_NOTICE("You start to cut [target.caste_type] carapace apart using \the [tool], carefully, with barely any acid."),
```

Comment:
there's tons of these and i'm not gonna make you change them, but \the [tool] is exactly the same as just doing [tool] and is preferred. but you don't gotta

Reply:
noted, why it was there in the first place then /shrug